okapi-core/src/main/resources/log4j2-json.properties

```diff
@@ -0,0 +1,38 @@
+status = error
```

Comment:
Can you mention that Okapi supports JSON logging and add a link to log4j2-json.properties?

Reply:
... in guide.md?

Reply:
Good idea. Will do